contracts/upgradeable_contracts/omnibridge_nft/BasicNFTOmnibridge.sol

```diff
@@ -157,77 +173,128 @@ abstract contract BasicNFTOmnibridge is
      * in order to avoid an attempt to steal the funds from a token with double addresses.
      * @param _token address of the token contract.
      * @param _receiver the address that will receive the token on the other network.
-     * @param _tokenId unique id of the bridged token.
+     * @param _tokenIds unique ids of the bridged tokens.
      */
-    function fixMediatorBalance(
+    function fixMediatorBalanceERC721(
         address _token,
         address _receiver,
-        uint256 _tokenId
+        uint256[] calldata _tokenIds
```

Comment:
It should work fine here. All other methods are adopted to operate with arrays. If more than one token id is passed here, then more than one token will be minted on the other side.
Since underlying communication protocol between mediators uses arrays, I think that there is no reason to specifically limit it to only one element.